flight_phase_files/HALO_RF02_20200122_info.yaml

```
@@ -24,59 +24,125 @@ remarks:
 - Descending Cirrus (9-13.5km) coming from West and South of the circle.
 - Away from the southwest edge of the track the air was almost entirely cloud free.
 segments:
-- kind: circle
-  name: c1
-  start: 2020-01-22 15:18:17
-  end: 2020-01-22 16:13:02
-- kind: circle_break
-  name: cb1to2
-  start: 2020-01-22 16:13:02
-  end: 2020-01-22 16:31:19
-- kind: circle
-  name: c2
-  start: 2020-01-22 16:31:19
-  end: 2020-01-22 17:26:29
-- kind: circle_break
-  name: cb2to3
-  start: 2020-01-22 17:26:29
-  end: 2020-01-22 17:43:08
-- kind: circle
-  name: c3
-  start: 2020-01-22 17:43:08
-  end: 2020-01-22 18:38:02
-- kind: baccardi_calibration
-  name: baccardi_cal
+- kinds:
+  - circle
+  name: Circle 1
+  irregularities:
+  - Circle is  363.52 deg to accomodate all 12 dropsondes
+  segment_id: HALO-0122_c1
+  start: 2020-01-22 15:17:17
+  end: 2020-01-22 16:13:05
+  good_dropsondes: 12
+- kinds:
+  - circle_break
+  name: Break between circle 1 and 2
+  irregularities: []
+  segment_id: HALO-0122_cb1
+  start: 2020-01-22 16:13:05
+  end: 2020-01-22 16:30:19
+- kinds:
+  - circle
+  name: Circle 2
+  irregularities: []
+  segment_id: HALO-0122_c2
+  start: 2020-01-22 16:30:19
+  end: 2020-01-22 17:26:44
+  good_dropsondes: 12
+- kinds:
+  - circle_break
+  name: Break between circle 2 and 3
+  irregularities: []
+  segment_id: HALO-0122_cb2
+  start: 2020-01-22 17:26:44
+  end: 2020-01-22 17:42:08
+- kinds:
+  - circle
+  name: Circle 3
+  irregularities: []
+  segment_id: HALO-0122_c3
+  start: 2020-01-22 17:42:08
+  end: 2020-01-22 18:38:57
+  good_dropsondes: 11
+- kinds:
+  - baccardi_calibration
+  name: Baccardi calibration
+  irregularities: []
+  segment_id: HALO-0122_bc1
   start: 2020-01-22 18:54:07
   end: 2020-01-22 19:08:55
```

To my knowledge, the interesting thing about the baccardi calibration pattern are the four straight legs of the box. To cover these, the time must be adjusted.
```suggestion
  start: 2020-01-22 18:56:20
  end: 2020-01-22 19:12:00
```
Note that this creates an overlap with `HALO-0122_sl1`, which should not be a problem though.

---

yes, that's true. I started the calibration a little later (18:56:46) as before, HALO was still rolling a bit